src/Stepper/StepConnector.js

```diff
@@ -0,0 +1,53 @@
+import React, {PropTypes} from 'react';
+import pure from 'recompose/pure';
+
+const StepConnector = (props, context) => {
+  const {muiTheme, stepper} = context;
```

Comment:
Shouldn't `stepper` be coming from `muiTheme` rather than directly from `context`?